sos/plugins/nvme.py

```diff
@@ -0,0 +1,119 @@
+### This program is free software; you can redistribute it and/or modify
+## it under the terms of the GNU General Public License as published by
+## the Free Software Foundation; either version 2 of the License, or
+## (at your option) any later version.
+
+## This program is distributed in the hope that it will be useful,
+## but WITHOUT ANY WARRANTY; without even the implied warranty of
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+## GNU General Public License for more details.
+
+## You should have received a copy of the GNU General Public License
+## along with this program; if not, write to the Free Software
+## Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+
+from sos.plugins import Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin
+import os, subprocess
+
+class NvmePlugin(Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin):
```

Comment:
Do not use `Plugin` suffix to class names. Just `Nvme` is fine (and must match the name given to `plugin_name`.